rskj-core/src/test/java/co/rsk/peg/ReleaseTransactionBuilderTest.java

```diff
@@ -390,6 +388,90 @@ public void buildEmptyWalletTo_utxoProviderException() throws InsufficientMoneyE
         Assert.assertFalse(result.isPresent());
     }
 
+    @Test
+    public void test_BuildBatchedPegouts_ok() throws UTXOProviderException, InsufficientMoneyException {
```

Comment:
I think it would be nicer to store the recipients in variables to make it clear when asserting the pegout transaction.

Aside from it, I don't follow the idea of the "one", "two", "three" values... could you clarify what's that?

Reply:
Resolved